src/controllers/settings.js

```diff
@@ -138,39 +132,51 @@ async function resendEmail (req, res) {
     throw fluentError('user-verify-token-error')
   }
 
-  await sendVerificationEmail(emailId)
+  await sendVerificationEmail(res, emailId)
```

Comment:
OK I made this less repetitive, but this approach makes `sendVerificationEmail` take `res` as an argument and it's now responsible for reporting success or failure... it seems like we must be able to have a better pattern than this though.

Reply:
yes.. not sure if this is better or worse...

Reply:
Thanks, I'll research this a bit on Monday... there's probably a better way to do this that I'm not thinking of.

Reply:
Oh, after you pointed out the existence of that `RateLimitError` class, I realized we could use that here as well - if the controller doesn't catch the exception from `sendVerificationEmail()`, then the default error handler does the right thing.

I put just the success JSON response back in. I bet we could simplify these too somehow, but I like the idea of the API route functions being in charge of the `req` and `res` objects and not delegating that to a function.